mapping/sustainability.yaml

```
@@ -289,10 +284,10 @@
 
         If an environmental impact assessment was conducted:
 
-        - Set `.environment.hasImpactAssessment` to `true`.
-        - Add a `Classification` object to the `.environment.impactCategories` array, set its `.scheme` to "ifc-environmental-social" and set its `.id` to the letter for the category into which the project falls.
+        1. Set [`.environment.hasImpactAssessment`](project-schema.json,,environment/hasImpactAssessment) to `true`.
+        2. Add a [`Classification`](../../reference/schema.md#classification) object to the [`.environment.impactCategories`](project-schema.json,,environment/impactCategories) array, set its [`.scheme`](project-schema.json,/definitions/Classification,scheme) to "ifc-environmental-social" and set its [`.id`](project-schema.json,/definitions/Classification,id) to the lowercase letter of the category into which the project falls.
```

```suggestion
        2. Add a [`Classification`](../../reference/schema.md#classification) object to the [`environment.impactCategories`](project-schema.json,,environment/impactCategories) array, set its [`.scheme`](project-schema.json,/definitions/Classification,scheme) to 'ifc-environmental-social' and set its [`.id`](project-schema.json,/definitions/Classification,id) to the lowercase letter of the category into which the project falls.
```

The other codes so far have all used single quotes with double quotes being used for set text so I'm just going with that convention now